python/nav/web/templates/seeddb/_filter_form_columns.html

```diff
@@ -0,0 +1,24 @@
+            <form class="{{attrs.form_class }}" method="{{ attrs.method }}"{% if attrs.action %} action="{{ attrs.action }}"{% endif %}>
+              <fieldset>
+                <legend>{{ attrs.legend }}</legend>
+                <div class="row">
+                  {% for field in filter_form %}
+                    <div class="columns medium-3">
+                      <div id="div_id_{{ field.name }}" class="ctrlHolder{% if field.errors %} error {% endif %}">
+                        {{ field.label_tag }}
+                        {{ field }}
+                        {% for error in field.errors %}
+                        <small id="error_{{ forloop.counter}}_id_{{ field.name }}" class="error">
+                          {{ error }}
+                        </small>
+                        {% endfor %}
+                      </div>
+                    </div>
+                  {% endfor %}
+                  <div class="columns medium-3">
+                    <label>&nbsp;</label>
+                    <input type="submit" name="submit" value="{{ attrs.submit_value }}" class="submit button postfix" id="submit-id-submit">
+                  </div>
+                </div>
+              </fieldset>
+            </form>
```

Comment:
This is so similar to the template @hmpf wrote for the filter form with a single drop-down that we should really look into merging the two templates, IMHO.  I doesn't have to be in this PR, but then we should make an issue for it so as not to forget to look into it.

Reply:
I added #2995.